taxi/views.py

```diff
@@ -87,3 +92,41 @@ class DriverListView(LoginRequiredMixin, generic.ListView):
 class DriverDetailView(LoginRequiredMixin, generic.DetailView):
     model = Driver
     queryset = Driver.objects.all().prefetch_related("cars__manufacturer")
+
+
+class DriverCreateView(LoginRequiredMixin, CreateView):
+    model = get_user_model()
+    fields = [
+        "username",
+        "first_name",
+        "last_name",
+        "license_number",
+        "password"
```

Comment:
When creating a new user, ensure that the password is hashed before saving the user instance. Using `set_password()` method on the user instance is recommended to handle password hashing.